_posts/2022-12-12-side-projects-2022.md

```diff
@@ -0,0 +1,172 @@
+---
+layout: post
+id: E629FACC-8499-4402-B03B-0F5DB13396BD
+title: The Sparking Joy of Side-Projects, 2022 Edition
+author: Mick F
+excerpt: >-
+  Reviewing 2022, the benefits I gained from running a newsletter as a
+  side-project are tremendous. Let me explain why.
+category: Journaling
+tags:
+  - Swift
+  - iOS
+  - JavaScript
+  - React
+---
+
+🎄 December is here! Like every year, it is time to watch _Home Alone_ again and
```

Comment:
```suggestion
🎄 December is here! Like every December, it is time to watch _Home Alone_ again and
```

Reply:
I didn't want to repeat `December`. What is wrong with `year`? What else can I say to not repeat `December`?